pyqtorch/primitives/parametric.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         self,
         generator: str | Tensor,
         qubit_support: int | tuple[int, ...] | Support,
-        param_name: str | int | float | torch.Tensor = "",
+        param_name: str | int | float | torch.Tensor | ConcretizedCallable = "",
```

Comment:
What I meant is that you added the support for passing a `ConcretizedCallable` directly as the parameter, as you write there in the type hint. You also added it above in the `Scale`. When I tried it, the scale worked fine, but the `Parametric` had some error that I don't recall exactly what it was. But since you added it for both, there should be a test for it.

Reply:
Removed the `ConcretizedCallable` from `parametric.py`

Reply:
restored